test/suites/integration/bsp/submit-proofs.test.ts

```diff
@@ -355,8 +359,14 @@ describeBspNet(
       const challengePeriod = challengePeriodResult.asOk.toNumber();
       // Then we calculate the next challenge tick.
       const nextChallengeTick = lastTickBspTwoSubmittedProof + challengePeriod;
-      // Finally, advance to the next challenge tick.
-      await userApi.advanceToBlock(nextChallengeTick);
+
+      const currentBlock = await userApi.rpc.chain.getBlock();
+      const currentBlockNumber = currentBlock.block.header.number.toNumber();
+
+      if (nextChallengeTick < currentBlockNumber) {
+        // Advance to the next challenge tick if needed
+        await userApi.advanceToBlock(nextChallengeTick);
+      }
```

Comment:
This should be the other way around. We should advance if `nextChallengeTick > currentBlockNumber`.